resources/views/fields/input.blade.php

```diff
@@ -1,5 +1,8 @@
 <div data-controller="input"
      data-input-mask="{{$mask ?? ''}}"
+     @class([
```

Comment:
I think this is a useful PR 👍
But maybe it would be better to add this directly in the class constructor:

```php
$this->beforeRender(function () {
    if ($this->get('type') === 'hidden' || $this->get('hidden', false)) {
        $this->hiddenFormType();
    }
});
```

This way, we’ll handle it in a more universal and elegant way.

Reply:
> I think this is a useful PR 👍 But maybe it would be better to add this directly in the class constructor:
> 
> ```
> $this->beforeRender(function () {
>     if ($this->get('type') === 'hidden' || $this->get('hidden', false)) {
>         $this->hiddenFormType();
>     }
> });
> ```
> 
> This way, we’ll handle it in a more universal and elegant way.

lgtm, im updated PR